sourcegraph/learn
# Java tutorial on ArrayOutofBoundsException

## Type of change
Tutorial

### What should this PR do?
Add a tutorial in Java for ArrayOutOfBounsdException as per https://github.com/sourcegraph/learn/issues/250

### What are the acceptance criteria? 
* Add a new file in the `posts` directory using the correct naming convention.
* The post should have the correct preamble at the top of the file
* Content of the post should follow the same format as https://learn.sourcegraph.com/how-to-troubleshoot-python-zerodivisionerror
* The author should be added to the `authors.yaml` (optional)

### How should this PR be tested?
* The post can be viewed locally by running `npm run dev` and navigating to the post at `http://localhost:3000/how-to-troubleshoot-java-arrayindexoutofboundsexception`
* Check that the post has correct spelling and grammar

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
